core/src/components/card/card.stories.tsx

```diff
@@ -50,22 +50,22 @@ export default {
         type: 'text',
       },
     },
-    headerImg: {
-      name: 'Header image',
-      description: 'Toggles an image in the header.',
+    thumbnail: {
+      name: 'Header thumbnail',
+      description: 'Toggles a thumbnail in the header.',
       control: {
         type: 'boolean',
       },
     },
-    headerPlacement: {
-      name: 'Header placement',
-      description: 'Sets the placement of the header, above or below the body image.',
+    imagePlacement: {
+      name: 'Body Image placement',
```

Comment:
```suggestion
      name: 'Body image placement',
```

Reply:
To me it feels like the "Image" should be spelled with lowercase letters, but maybe you don't agree?